.github/workflows/atlas.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           PATTERNS: |
             x/auth/atlas/**
-      - uses: marbar3778/atlas_action@main
+      - uses: tac0turtle/atlas_action@main
```

Comment:
Can we actually delete this action. Should figure out how to push atlas but for now its not needed

Reply:
Sure!